taskcluster/docker/ui-tests/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM $DOCKER_IMAGE_PARENT
+
+LABEL authors="Richard Pappalardo <rpappalax@gmail.com>, Aaron Train <atrain@mozilla.com>"
+LABEL maintainer="Richard Pappalardo <rpappalax@gmail.com>"
+
+#----------------------------------------------------------------------------------------------------------------------
+#-- Test tools --------------------------------------------------------------------------------------------------------
+#----------------------------------------------------------------------------------------------------------------------
+
+RUN apt-get install -y jq \
+    && apt-get clean
+
+USER worker:worker
+
+ENV GOOGLE_SDK_DOWNLOAD ./gcloud.tar.gz
+ENV GOOGLE_SDK_VERSION 233
+
+ENV TEST_TOOLS /builds/worker/test-tools
+ENV PATH ${PATH}:${TEST_TOOLS}:${TEST_TOOLS}/google-cloud-sdk/bin
+
+RUN mkdir -p ${TEST_TOOLS} && \
+    mkdir -p ${HOME}/.config/gcloud
+
+RUN curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}.0.0-linux-x86_64.tar.gz --output ${GOOGLE_SDK_DOWNLOAD} \
+    && tar -xvf ${GOOGLE_SDK_DOWNLOAD} -C ${TEST_TOOLS} \
+    && rm -f ${GOOGLE_SDK_DOWNLOAD} \
+    && ${TEST_TOOLS}/google-cloud-sdk/install.sh --quiet \
+    && ${TEST_TOOLS}/google-cloud-sdk/bin/gcloud --quiet components update
+
+# Flank v21.07.1
+
+RUN URL_FLANK_BIN="$($CURL --silent 'https://api.github.com/repos/Flank/flank/releases/latest' | jq -r '.assets[] | select(.browser_download_url | test("flank.jar")) .browser_download_url')" \
```

Comment:
Mind replacing `latest` with `48276753` (the previous release) – since this Docker will be recreated, this will download the latest version of Flank with `latest` and it has an issue I'm familiar with. I recently replaced this in Fenix. https://github.com/mozilla-mobile/fenix/commit/bca177541e8e16708fa3bd83c89a8dadc5064094